player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```diff
@@ -156,11 +156,10 @@ endPage.controller("endPageController", function($scope, $rootScope, $state,$ele
                 org.ekstep.contentrenderer.initializePreview(object)
                 EkstepRendererAPI.dispatchEvent('renderer:player:show');
         } else {
-            if(globalConfig.deeplinkBasePath){
-                var deepLinkURL = globalConfig.deeplinkBasePath + "c/" + contentToPlay.content.identifier + "?hierarchyInfo=" + JSON.stringify($rootScope.content.hierarchyInfo);
-                window.open(deepLinkURL, "_system");
+            if(contentMetadata.identifier) {
+                window['playerActionHandlerDelegate'].onContentNotFound(contentMetadata.identifier, contentMetadata.hierarchyInfo);
```

Comment:
window['playerActionHandlerDelegate']
Discuss this before code change.

Reply:
This is the callback for the mobile, for the deep linking. This will execute if there is no content available.